gwml2_schema.sql

```diff
@@ -0,0 +1,115 @@
+CREATE SCHEMA groundwater;
+
+CREATE TYPE groundwater.CI_OnLineFunctionTerm AS ENUM ('download', 'information', 'offlineAccess', 'order', 'search');
```

Comment:
We should only use declared types / ENUMS when we are 100% sure that there will be no need to extend the list of allowed values. Generally I think it is better to model these as entity relations